Lecture1/Task1/com/playtika/Phrase.java

```diff
@@ -0,0 +1,30 @@
+package com.playtika;
+import java.util.Scanner;
+import java.util.Arrays;
+
+public class Phrase {
+    public static void main( String[] args ) {
+        Scanner in = new Scanner(System.in);
+        System.out.println("Enter something: ");
+        String input = in.nextLine();
+
+        System.out.println("Reverse: " + new StringBuilder(input).reverse().toString()); //task 1.1
+
+        String[] words = input.split(" ");
+        System.out.println ("Split:");
+        for (String word : words) {
+            System.out.println(word); // task 1.2
+        }
+        System.out.println("Change: " + input.replace(' ', '*'));// task 1.3
+        String strUpper = input.toUpperCase();
+        System.out.println ("Upper case: " +strUpper);// task 1.4
+        input.length();
+        if (input.length () < 10)
```

Comment:
I assume we can declare the max index as a variable above and reuse it in two places